src/main/java/org/folio/rest/impl/Processor.java

```diff
@@ -479,68 +492,97 @@ private ProcessedRule processRule(JsonObject rule, Leader leader, String data, S
       //for example:
       //          "type": "custom",
       //          "value": "DATA.replace(',' , ' ');"
-      String[] functions = condition.getString(TYPE).split(",");
-      //we need to know if one of the functions is a custom function
-      //so that we know how to handle the value field - the custom indication
-      //may not be the first function listed in the function list
-      //a little wasteful, but this will probably only loop at most over 2 or 3 function names
-      for (String function : functions) {
-        if(CUSTOM.equals(function.trim())){
-          isCustom = true;
-          break;
-        }
-      }
+      String[] functions = ProcessorHelper.getFunctionsFromCondition(condition);
+      isCustom = checkIfAnyFunctionIsCustom(functions, isCustom);
```

Comment:
Writing it this way shows more clearly what happens:
`isCustom = isCustom || hasCustomFunction(functions)`